constraints.txt

```diff
@@ -1,10 +1,10 @@
-Jinja2==2.11.2
+Jinja2==2.11.3
 MarkupSafe==1.1.1
-PyYAML==5.3.1
-cffi==1.14.4
+PyYAML==5.4.1
+cffi==1.14.5
 coverage==4.5.4
-cryptography==3.2.1
-packaging==20.7
+cryptography==3.4.6
```

Comment:
To avoid conflicts with ansible-test this should be limited to < 3.4. That upper bound should be added to the requirements.txt file. See: https://github.com/ansible/ansible/blob/3e1f6484d77f2d7546952cfa22a8534d74ed3dc6/test/lib/ansible_test/_internal/executor.py#L275-L277

Reply:
alright, was wondering why there was not constraint :)